tests/trading/test_orders.py

```diff
@@ -23,207 +23,215 @@
 WORSE_ORDER_ID = 6
 GAS_PRICE = 7
 
+ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+
 def test_walkOrderList_bids(contractsFixture):
     market = contractsFixture.binaryMarket
-    orders = contractsFixture.contracts['orders']
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
+    orders = contractsFixture.contracts['Orders']
+    ordersFetcher = contractsFixture.contracts['OrdersFetcher']
     outcomeID = 1
     order = {
         "orderID": 5,
```

Comment:
This should be `'orderId': longTo32Bytes(5),`.  You'll have to create a method for `longTo32Bytes`, there are some helpful tips on how to do this on StackOverflow.

Reply:
Alternatively, a randomly generated 32-byte array would serve the purposes here.  It just needs to be unique.

Reply:
Same comment for other tests in this file.